app/models/gestionnaire.rb

```diff
@@ -44,6 +50,10 @@ def follow? dossier_id
     Follow.where(gestionnaire_id: id, dossier_id: dossier_id).any?
   end
 
+  def assigned_on_procedure?(procedure_id)
+    procedures.find_by(id: procedure_id).present?
```

Comment:
Hmm, il semble que `find_by` soit `the way to go` :

> Dynamic attribute-based finders are a mildly deprecated way of getting (and/or creating) objects by simple queries without turning to SQL. They work by appending the name of an attribute to find_by_ like Person.find_by_user_name. Instead of writing Person.find_by(user_name: user_name), you can use Person.find_by_user_name(user_name).

http://api.rubyonrails.org/classes/ActiveRecord/Base.html